source/Octopus.Client.Tests/Integration/OctopusClient/BuildServerHeadersAzureDevOpsTests.cs

```diff
@@ -0,0 +1,11 @@
+using Octopus.Client.Model;
+
+namespace Octopus.Client.Tests.Integration.OctopusClient
+{
+    public class BuildServerHeadersAzureDevOpsTests : BuildServerHeadersBaseTests
+    {
+        protected override string EnvironmentVariableName => OctopusCustomHeaders.EnvVar_AzureDevOps;
```

Comment:
Potentially coupled too closely to the implementation here - ie, the value could change and the test would pass, but it'd break when parsing the telemetry.

Reply:
Yep, good point, I'll change that